jabgui/src/main/java/org/jabref/gui/walkthrough/Timeout.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.gui.walkthrough;
+
+import javafx.animation.PauseTransition;
+import javafx.util.Duration;
+
+/// Executes a runnable after a specified duration. It uses a [PauseTransition] to
+/// ensure the runnable is executed on the JavaFX Application Thread.
```

Comment:
PauseTransition is not automatically run in JavaFX thread just because it sits in javafx package. You need to run it in Platform.runLater()

Reply:
Use UiTaskExecutor::runInJavaFxThread

Reply:
Oversaw something. Its fine.